### DISCUSSION TITLE
Librem Phone version?

### DISCUSSION THREAD

**Discussion by programmin1** [iSoron/uhabits#711] [Feature Requests]
I saw there was a ios request... Why don't you port this app to Linux (Librem phone, Pinephone etc.?)

That is a more OSS friendly platform. And soon a lot more people are going to be switching to Librem phone as they ship soon.

**Comment by iSoron** [comment]
Thank you for the suggestion, @programmin1. If we had a GTK+ desktop version of Loop Habit Tracker, would you expect it to run essentially unmodified on these phones?

**Reply by programmin1** [reply]
Yes, Librem/Pinephone run standard GTK desktop apps. I actually wrote a 8 part blog post series on how I built a Linux-desktop/Libremphone app (Repeater-START) here and this part details testing and adjusting for a mobile screen:
https://howtotrainyourrobot.com/building-an-amateur-radio-app-for-librem-phone-part-3/

**Comment by hiqua** [comment]
In the meantime, Linux phone users can try to use anbox and report problems. I've tried it quickly, and it seems to work fine.

**Comment by 12people** [comment]
There's a different free and open-source habit tracker that could be ported to Linux: [Habo](https://github.com/xpavle00/Habo), which is a GPL-licensed multi-platform Flutter app.

It's only on Android and iOS for now, but I'm working on a pull request to bring it to Linux too.

**Reply by 12people** [reply]
And here's another one: [Table Habits](https://github.com/FriesI23/mhabit)
